news/_posts/translation/2021-06-03-do-you-speak-prestashop-may-2021.md

```diff
@@ -0,0 +1,116 @@
+| 4. | mirmal | Esperanto | 384
+| 5. | Begi Ganellari (begiganellari1997) | Albanian | 349
+| 6. | Сергій Мацапура (mazapura) | Ukrainian | 272
+| 7. | M. Mirena (mmirena) | Albanian | 232
+| 8. | A. L (roynsdx) | Arabic | 210
+| 9. | Ahmad Abu Ashour (gpsarab) | Arabic | 110
+| 10. | Gabriel Tenita (ggedamed) | Romanian | 109
+ 
+#### Top 5 proofreaders in May:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | SNor42 | Hungarian | 3,134
+| 2. | Girdman.com (anargaragezov) | Azerbaijani | 1,665
+| 3. | M. Mirena (mmirena) | Albanian | 414
+| 4. | A. L (roynsdx) | Arabic | 344
+| 5. | Gabriel Tenita (ggedamed) | Romanian | 94
+
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of May, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **18 languages**: 
+ 
+| Bosnian | Chinese traditional | Danish | English |
+| Finnish | French |Greek | Hebrew | 
+| Italian | Lithuanian | Persian | Portuguese, Brazil | 
+| Romanian | Serbian | Slovak | Slovene | 
+| Spanish, Spain | Swedish |
+
+Thank you all for your contribution! 
+
+#### Almost fully translated languages 
+
+Korean, Polish, and Czech are 99% translated. Almost there! :muscle: 
+If you would like to contribute to these projects, do not hesitate to [join us on Crowdin](https://crowdin.com/project/prestashop-official). 
+
+Note that Turkish, that appeared in this category last month, was 100% translated at the end of May. That is great, thank you to all the Turkish translators!
+
+## Languages with the best evolution
+
+The following languages had the best progress thanks to the translation community:
+ 
+* **Hungarian** (+6% to reach 83% validated) 
+* **Azerbaijani** (+2% to reach 36% validated)
+* **Estonian** (+2% to reach 100% translated :tada: )
+
+Congratulations! :muscle:
+ 
+![Best translation progress in May 2021](/assets/images/2021/06/build-crowdin-progress-may21.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since it is working on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
+
+If you have not joined us on Crowdin yet, [it is never too late](https://crowdin.com/project/prestashop-official)! :wink:
+ 
+If you want to gather your fellow translators to work towards a better harmonization, start a glossary, or anything else, do let me know: I will include a word about it in the next monthly report.
```

Comment:
If you wish to, we could create a #translation Slack channel for translators to discuss? (in the Slack Open Source)

Reply:
I think that would be great! :-)